dev/com.ibm.websphere.appserver.features/visibility/public/crac-1.0/resources/l10n/io.openliberty.crac-1.0.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables the CRaC APIs. 
```

Comment:
Is this another temporary feature that will get absorbed into the kernel before GA? If not temporary then the description should be more verbose ... spell out "Coordinated Restore at Checkpoint"

Reply:
I doubt it will be folded into the kernel as it is providing third-party API.  I can update the description.

Reply:
And the copyright date then ;-)